### DISCUSSION TITLE
Field name to display the tax percentage in the quotation & Invoice PDF

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#17585] [How to]
Hi,

I am using the below syntax to display the product table in the quotation module.

$(inventory : type=table columns=seq,comment1,qty,unit,total,tax , href=no , labels=SNo,Product Description,Qty,Unit,Product Price, Tax , width=10px,225px,50px,45px,105px,95px )$

Cloud you please share the field name to display the  tax percentage in this table.

Thanks & Regards.
Ilamparithi

**Comment by rskrzypczak** [comment]
![image](https://github.com/user-attachments/assets/f59978ad-6e68-48c3-a719-ff00311b9447)
